.circleci/config.yml

```diff
@@ -4,11 +4,22 @@ orbs:
   architect: giantswarm/architect@2.7.0
 
 workflows:
-  test:
+  build:
     jobs:
-      - architect/go-test:
-          name: go-test
+      - architect/go-build:
+          name: go-build
+          binary: crd-installer
+          filters:
+            tags:
+              only: /^v.*/
+
+      - architect/push-to-docker:
+          name: push-to-quay
+          image: "quay.io/giantswarm/crd-installer"
+          username_envar: "QUAY_USERNAME"
```

Comment:
Why not use the architect context?

Reply:
Because I copied it from kvm-operator and forgot about contexts :) I'll update it.